diff --git a/node_modules/@glidejs/glide/package.json b/node_modules/@glidejs/glide/package.json
index cf06f9e..fc4afb4 100755
--- a/node_modules/@glidejs/glide/package.json
+++ b/node_modules/@glidejs/glide/package.json
@@ -37,6 +37,7 @@
     "lint": "eslint {src,tests}/**/*.js"
   },
   "exports": {
+    "./src/*": "./src/*",
     ".": {
       "import": "./dist/glide.esm.js",
       "require": "./dist/glide.js",
diff --git a/node_modules/@glidejs/glide/src/components/html.js b/node_modules/@glidejs/glide/src/components/html.js
index e04eb23..bc3c2c4 100644
--- a/node_modules/@glidejs/glide/src/components/html.js
+++ b/node_modules/@glidejs/glide/src/components/html.js
@@ -1,5 +1,5 @@
 import { warn } from '../utils/log'
-import { exist, toArray } from '../utils/dom'
+import { toArray } from '../utils/dom'
 import { define } from '../utils/object'
 import { isString } from '../utils/unit'
 
